### PR TITLE
Feature/forward proxy and web server

### PR DESCRIPTION
**Pull request by skovbakke**
**Description of your changes:**
We have set up the forward proxy service. It is able to handle the requests for git repositories, helm charts, and container images. It needs the artifact-registry and git-server services running in the same namespace in order to function. 

Checklist:

* [x] I have bumped the chart version
* [x] I have documented my changes if this is needed
* [x] I have described my changes in the "description of your changes" field
* [x] I have tested this change on a running cluster and the Argocd dashboard shows healthy and synced
* [ ] I have created the necessary tests in the chart, if they are possible/necessary
* [x] I have squashed commits if necessary

### CODE REVIEW

File: charts/fwd-proxy/chart/templates/configmap-example-html.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```
Comment (jacobcrawford) [inline charts/fwd-proxy/chart/templates/configmap-example-html.yaml:1]: Is this file unused?

File: charts/fwd-proxy/chart/config/demo.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+static_resources:
```
Comment (jacobcrawford) [inline charts/fwd-proxy/chart/config/demo.yaml:1]: Is this file unused?

File: charts/fwd-proxy/chart/test/example.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (jacobcrawford) [inline charts/fwd-proxy/chart/test/example.html:1]: is this file unused?

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
Seems like the pod is killed by its own livenessprobe before it fully starts up. A delay or longer intervals might fix this:
image

**Comment by klausenbusk** [comment]
> Seems like the pod is killed by its own livenessprobe before it fully starts up. A delay or longer intervals might fix this:

The service has a hard requirement on the artifact-service and git-service. Was they running when you tested?

**Comment by skovbakke** [comment]
image
Cannot reproduce the failure @jacobcrawford. It works as expected.

**Comment by jacobcrawford** [comment]
> > Seems like the pod is killed by its own livenessprobe before it fully starts up. A delay or longer intervals might fix this:
> 
> The service has a hard requirement on the artifact-service and git-service. Was they running when you tested?

If there is a hard requirement it should be documented in the README, else the char should be able to be deployed by itself?

**Review by jacobcrawford — APPROVED** [review]
LGTM. Unless you wanted to squash the commits further, this is ready for merge :)

**Pull request merged**